device/bgp/bgpsrtebindingsubtlv.yaml

```diff
@@ -2,12 +2,12 @@ components:
   schemas:
     BgpSrte.BindingSubTlv:
       description: >-
-        Configuration for BGP binding sub TLV settings.
-      type: object
+        Configuration for the binding SID sub-TLV is used to signal 
```

Comment:
Configuration for the binding SID sub-TLV. This is used to signal the binding SID related information of the SR Policy candidate path.